xml/gl.xml

```diff
@@ -20738,14 +20738,14 @@ typedef unsigned int GLhandleARB;
             <param group="MapTarget"><ptype>GLenum</ptype> <name>target</name></param>
             <param group="MapQuery"><ptype>GLenum</ptype> <name>query</name></param>
             <param><ptype>GLsizei</ptype> <name>bufSize</name></param>
-            <param><ptype>GLdouble</ptype> *<name>v</name></param>
+            <param len="COMPSIZE(bufSize)"><ptype>GLdouble</ptype> *<name>v</name></param>
```

Comment:
Same comment as in #438

Reply:
This is intentional and will not result in a merge conflict.

Reply:
From #438:

> COMPSIZE isn't really needed for single-parameter counts, mainly because there's nothing to compute. This is just an inconsistency in some places that the best solution for fixing would be removing unnecessary COMPSIZE from single parameters, rather than adding them elsewhere.
> 
> COMPSIZE is only needed where multiple parameters are used as part of the length calculation

Reply:
I don't think that's true everywhere although it may be mostly true. I believe there's a couple of places where a count is given in units different than the underlying type. What COMPSIZE indicates is that additional logic is needed to compute the actual size from the parameters. That said we use COMPSIZE for exactly nothing; it is a 30 year holdover from SGI's original .spec files and the actual logic required always had to be externally defined. So I don't really care whether it stays or goes.